### PR TITLE
Support additional content reader types

### PR DESCRIPTION
**Pull request by jacksonh**
- Implement a reader for epubs
- Add epub container

### CODE REVIEW

File: packages/api/src/services/speech.ts [hunk 1]
```diff
@@ -8,7 +10,10 @@ export const shouldSynthesize = async (
   userId: string,
   page: Page
 ): Promise<boolean> => {
-  if (page.pageType === PageType.File || !page.content) {
+  if (
+    contentReaderForPageType(page.pageType) === ContentReader.Web ||
```
Comment (sywhb) [inline packages/api/src/services/speech.ts:14]: I think this should be `!==`

**Review by sywhb — COMMENTED** [review]
I think only the email attachment change is missing. I can add it to your branch too

File: packages/api/src/services/speech.ts (hunk 1 above)
Comment (jacksonh) [reply packages/api/src/services/speech.ts]: OMG that would have been hard to catch in testing too

**Comment by jacksonh** [comment]
> I think only the email attachment change is missing. I can add it to your branch too

yeah, we could probably do separately also, I think we will also need to create a `epub-handler` to do the search indexing

**Comment by sywhb** [comment]
@jacksonh yeah, `epub-handler` is another one.

Cool. LGTM

**Review by sywhb — APPROVED** [review]

**Comment by jacksonh** [comment]
Tagged this ticket which kind of covers an epub handler: https://github.com/omnivore-app/omnivore/issues/2008

**Comment by jacksonh** [comment]
Closes https://github.com/omnivore-app/omnivore/issues/2007 
Closes https://github.com/omnivore-app/omnivore/issues/2006

**Pull request merged**